woocommerce/woocommerce-android
# [Order Creation] handle order creation

Closes: #5640 

### Description
This PR adds support for displaying a "create" button when the order has products and customer data, and handles the creation logic.

There are additional changes just to improve the form:
1. I made the form scrollable to fix the landscape issue.
2. I added possibility to hide the edit button from the form's sections, even if they have content, this is needed for products.

### Testing instructions
1. Make sure order creation flag is enabled from the beta settings.
2. Go to order list, and click on the + FAB.
3. Add some products to the order.
4. Add a billing and shipping address.
5. Click on Create.
6. Confirm that a loading dialog is displayed.
7. After the order is created, confirm that you are taken to the order details to display the created order.

### Images/gif
<img width=400 src="https://user-images.githubusercontent.com/1657201/149371266-f1a0213a-74d3-4261-975f-da754bcaeb5e.gif"/>
